README.md

```diff
@@ -59,18 +59,19 @@ Note: The first time setting up will clone any submodules with git so the first
 The documentation can be found on the [OpenAssetTools.dev page](https://openassettools.dev).
 It is hosted on Github Pages and its source can be found in the [OAT-Docs Repository](https://github.com/Laupetin/OAT-Docs).
 
-For examples of mods that can be built with OAT you can either look into the [docs folder](docs/example) for some (currently very barebones) examples
+For examples of mods that can be built with OAT, you can either look into the [docs folder](docs/example) for some (currently very barebones) examples
 or check out one of the following projects that are using OAT:
 
 * [JezuzLizard/t6-fastfile-mods](https://github.com/JezuzLizard/t6-fastfile-mods)
 * [Jbleezy/BO2-Reimagined](https://github.com/Jbleezy/BO2-Reimagined)
+* [diamante0018/PlutoIW5Arena](https://github.com/diamante0018/PlutoIW5Arena) (Contains GitHub CI/CD pipeline for building fastfiles)
```

Comment:
"Includes GitHub actions pipeline building the mod" otherwise it might sound like that's the only thing included there?